modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/PagesWriteThrottle.java

```diff
@@ -155,6 +160,9 @@ public PagesWriteThrottle(PageMemoryImpl pageMemory,
                 LockSupport.parkNanos(throttleParkTimeNs);
 
             pageMemory.metrics().addThrottlingTime(U.currentTimeMillis() - startTime);
+
+            if (performanceStatsEnabled)
```

Comment:
Can we move this to `addThrottlingTime` inside page memory metrics?

Reply:
done